src/Carter/CarterExtensions.cs

```diff
@@ -191,6 +192,8 @@ private static void WireupCarter(this IServiceCollection services, DependencyCon
             }
 
             services.AddSingleton<IResponseNegotiator, DefaultJsonResponseNegotiator>();
+
+            services.AddSingleton<IModelBinder, DefaultJsonModelBinder>();
```

Comment:
Can you add a method GetModelBinders and do what GetResponseNegotiators does and finds binders and ignores the default one, that way people can just drop theres in